samples/tab-sso/nodejs/.gitignore

```diff
@@ -66,7 +66,6 @@ build/
 # # # # # # # # # # # # # # 
 
 # TeamsFx files
-env/.env.*
 .DS_Store
```

Comment:
Also please refer to https://github.com/formulahendry/teamsfx-migrator/blob/main/templates/tab-sso/.gitignore.example#LL68C1-L84C5

Reply:
reverted